device/globals/game.gd

```diff
@@ -115,10 +115,7 @@ func spawn_action_menu(obj):
 		return
 	action_menu.show()
 	var pos
-	if obj.has_node("action_menu_pos"):
```

Comment:
I think the purpose of this was to fix the action menu to a specific position on screen, using a `Position2D` node. I don't remember ever playing a game that used this feature though, so I don't know how useful it was.